.github/ISSUE_TEMPLATE/update-external-links-gallery.md

```diff
@@ -5,40 +5,42 @@ title: ''
 labels: 'external-links-gallery-submission'
 ---
 
-<!-- Please fill out the template below. -->
+<!-- Please fill out the template below by adding the necessary information in between the single quotes in each relevant or known field. -->
 
 Make sure the following items are specified :
 
-- `url`
-- `title`
-- `description`
+- [ ] `url`
```

Comment:
How are these checkboxes used? I.e. is the  issue submitter intended to check them, or do they get auto-populated somehow? 

In the case of the former (issue submitter checks them):

1. What happens if they are not checked? Any consequence, or are they just intended to help guide the submitter?
2. My understanding with template check boxes is that they can't be checked until **after** the issue is submitted, which is pretty confusing to the uninitiated :-). We should probably warn them of this.


If the checkboxes serve no purpose other than to remind the user I would suggest not having them given the awkward that GitHub supports them.

Reply:
The user checks them without any consequence, just works as a guide. I'll change them back.